host/sgx/windows/enter.asm

```diff
@@ -1,184 +0,0 @@
-;; Copyright (c) Open Enclave SDK contributors.
-;; Licensed under the MIT License.
-
-include ksamd64.inc
-
-extern __oe_dispatch_ocall:proc
-extern oe_save_host_context:proc
-extern oe_restore_host_context:proc
-
-;;==============================================================================
-;;
-;; void oe_enter(
-;;     [IN] void* tcs,
-;;     [IN] uint64_t aep,
-;;     [IN] uint64_t arg1,
-;;     [IN] uint64_t arg2,
-;;     [OUT] uint64_t* arg3,
-;;     [OUT] uint64_t* arg4,
-;;     [OUT] oe_enclave_t* enclave);
-;;
-;; Parameters passed on register and stack:
-;;     RCX      - tcs: thread control structure (extended)
-;;     RDX      - aep: asynchronous execution procedure
-;;     R8       - arg1
-;;     R9       - arg2
-;;     [RBP+48] - arg3
-;;     [RBP+56] - arg4
-;;     [RBP+64] - enclave
-;;
-;; These registers may be destroyed across function calls:
-;;     RAX, RCX, RDX, R8, R9, R10, R11
-;;
-;; These registers must be preserved across function calls:
```

Comment:
Consider retaining the documentation about registers preserved across function calls.

Reply:
Done.